src/databricks/labs/mcp/servers/unity_catalog/tools/vector_search.py

```diff
@@ -1,62 +1,78 @@
-import io
 import json
-from contextlib import redirect_stdout
+from typing import List
```

Comment:
please just use `list` it's supported for typing since some old version. Even the official typing doc recommends to use `list[SomeType]` as per https://docs.python.org/3/library/typing.html